aten/src/ATen/native/native_functions.yaml

```diff
@@ -2484,6 +2484,11 @@
   dispatch:
     CPU: dense_to_mkldnn
 
+- func: mkldnn_reorder_conv2d_weight(Tensor self, int[2] padding=0, int[2] stride=1, int[2] dilation=1, int groups=1) -> Tensor
+  variants: method
```

Comment:
do we need this to be a method?  can it just be a function in python_module nn?

Reply:
sure